.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: os-name
+    attributes:
+      label: OS Name
+      description: Which Operating System are you using?
+      options:
+        - Windows
+        - MacOS
+        - Linux
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: Operating System Version
+      description: |
+        Which version of your operating sytem do you use?
```

Comment:
For macOS specifically, we should ask if they use an Intel or Apple Silicon Mac. This info is included in the link you provided to the users.